From cf7793131945d30128115cc9bac9d1b67c258d6f Mon Sep 17 00:00:00 2001
From: Oliver Neukum <oneukum@suse.com>
Date: Thu, 1 Jun 2017 10:41:30 +0200
Subject: [PATCH 2/2] kABI: mask an include
References: bsc#994364
Patch-mainline: Never (kABI fix only)

The checker generates a false positive due to an added include

Signed-off-by: Oliver Neukum <oneukum@suse.com>
---
 drivers/base/power/trace.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/base/power/trace.c b/drivers/base/power/trace.c
index 35b2f2f1477c..14bdde49e92f 100644
--- a/drivers/base/power/trace.c
+++ b/drivers/base/power/trace.c
@@ -10,7 +10,9 @@
 #include <linux/resume-trace.h>
 #include <linux/export.h>
 #include <linux/rtc.h>
+#ifndef __GENKSYMS__
 #include <linux/suspend.h>
+#endif
 
 #include <asm/rtc.h>
 
-- 
2.12.3

